src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +159,45 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp         = set_bias(l0, 2, args);
+        auto w_zp          = set_bias(weights, 3, args);
+        auto is_quant_conv = opd.op_name == "quant_convolution";
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+
+        // Check for type mismatch on parse
```

Comment:
Do we check somewhere if the input type of L0 should be one of: int8, uint8. Same for Weights. Also, if the output should be int32 only? Thanks.

Reply:
Output should only be int32. I'm leaving the input to be checked by quant_convolution operator itself which checks weights and handles the proper output size of int32

https://github.com/ROCm/AMDMIGraphX/blob/develop/src/include/migraphx/op/quant_convolution.hpp#L91